File: C++/BOJ/1259.cpp
#include<iostream>

using namespace std;

int main()
{
	while (true)
	{
		bool	flag = true;
		string num;
		cin>>num;
		if (num == "0")
			break;
		else
		{
			int len = num.size();
			for (int i = 0; i < len; i++)
			{
				if (num[i] != num[len -1 -i])
					flag = false;
			}
		}
		if (flag == false)
			cout<<"no\n";
		else
			cout<<"yes\n";
	}
}

File: String/1259.py
import sys

while True:
    flag = 0
    num = sys.stdin.readline().strip()
    if num == "0":
        break
    else:
        for i in range(len(num)):
            if num[i] != num[-1 -i]:
                flag += 1
    if flag != 0:
        print("no")
    else:
        print("yes")
                
